src/components/toolbar/toolbarFieldRangedMonthly.js

```diff
@@ -0,0 +1,108 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { reduxTypes, store } from '../../redux';
+import { Select } from '../form/select';
+import { RHSM_API_QUERY_GRANULARITY_TYPES as FIELD_TYPES, RHSM_API_QUERY_TYPES } from '../../types/rhsmApiTypes';
+import { dateHelpers } from '../../common';
+import { translate } from '../i18n/i18n';
+
+/**
+ * Select field options.
+ *
+ * @type {{title: (string|Node), value: string, selected: boolean}[]}
+ */
+const toolbarFieldOptions = dateHelpers.getRangedMonthDateTime().listDateTimeRanges.map(dateTime => ({
+  ...dateTime,
+  selected: false
+}));
```

Comment:
note: may be better to fully leverage this helper method by implementing a look up similar to how we handle granularity, with a list/select populated by month names

Reply:
Attempted this, seems to balloon the code a little, requires handling "toISOString",  https://github.com/cdcabrera/curiosity-frontend/tree/20210224-dev-isostring-date-ent-3507